common/src/main/java/com/microsoft/identity/common/internal/platform/DevicePopManager.java

```diff
@@ -243,50 +237,34 @@
 
     DevicePopManager() throws KeyStoreException, CertificateException,
             NoSuchAlgorithmException, IOException {
-        mKeyStore = KeyStore.getInstance(ANDROID_KEYSTORE);
-        mKeyStore.load(null);
-        mKeyAlias = DEFAULT_KEYSTORE_ENTRY_ALIAS;
+        this(DEFAULT_KEYSTORE_ENTRY_ALIAS);
     }
 
     DevicePopManager(@NonNull final String alias) throws KeyStoreException, CertificateException,
             NoSuchAlgorithmException, IOException {
-        mKeyStore = KeyStore.getInstance(ANDROID_KEYSTORE);
-        mKeyStore.load(null);
-        mKeyAlias = alias;
+        String mKeyAlias = alias;
```

Comment:
```suggestion
        String keyAlias = alias;
```

(Removing `m` since no longer a member var)